homework_2.py

```diff
@@ -0,0 +1,27 @@
+eleks_employees = ["Nataly", "John", "Peter", "James", "Jacob"]
+toshiba_employees = ["Nelly", "Harry", "Jimmy", "Natan", "Mark"]
+new_toshiba_employees_list = (toshiba_employees + eleks_employees)
+eleks_employees.clear()
+print("Toshiba employees: " + str(new_toshiba_employees_list))
+print("Elex employees: " + str(eleks_employees))
+
+casino_blacklist = {"Nataly Parker", "John Smith", "Peter Prent", "James Jacob", "Jacob Hammer"}
+poker_blacklist = {"Nataly Parker", "Smith Janny", " Prent Harem", "James John", "Hammer Henthon"}
+alcohol_blacklist = {"Nataly Parker", "Perry Janny", "Oleh Harem", "Jerens John", "Hammy Henthon"}
+total_blacklist = casino_blacklist.intersection(poker_blacklist, alcohol_blacklist)
+print(total_blacklist)
+
+vegeterians_list = ["Natali"]
+omnivores_list = ["Alex"]
+omnivores_list.extend(vegeterians_list)
+print(omnivores_list)
+
+vip_box = ("Hanna", "Helen", "Joye")
+common_room = ["Peter", "Jeniffer"]
+common_room.append([])
+print(vip_box)
+print(common_room)
+
+group_of_people = frozenset(["John Dow", "Marta Dow", "John Dow"])
```

Comment:
you can not use set for this task.

Reply:
Only loops in my mind)
Fixed not universal code..